Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM frolvlad/alpine-java:jre8-slim
+EXPOSE 8000
+COPY target/covid19-1.0.0-SNAPSHOT/*.* ./
+ENTRYPOINT ["java","-jar","covid19.jar"]
+
```

Comment:
Hi @anjana-somathilake 

Can't we do the build too inside the docker file?

Reply:
we can,  then the docker should have maven installed as a dependency, building on the dev and deploying on the server is what makes sense for developers.

Reply:
do we have a CI service?